6238_prototype/src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.SmartDashboardParam;
+import frc.robot.subsystems.BallSubsystem;
+
+public class ShooterCommand extends CommandBase{
+    private final BallSubsystem ball; 
+    private final SmartDashboardParam upperShooterSpeedTarget;
+    public ShooterCommand(BallSubsystem ball) {
+        this.ball = ball;
+        addRequirements(ball);
+        upperShooterSpeedTarget = new SmartDashboardParam("upperMotorSpeedTarget");
+    }
+
+    public void execute() {
+        ball.extend();
+        double upperMotorSpeedTarget = upperShooterSpeedTarget.get();
+        double upperMotorSpeedCurrent = ball.getSpeedUpperMotor();
+        SmartDashboard.putNumber("upperMotorSpeedCurrent", upperMotorSpeedCurrent);
+        SmartDashboard.putNumber("upperMotorSpeedTarget", upperMotorSpeedTarget);
+        SmartDashboard.putNumber("upperMotorSpeedError",
+            upperMotorSpeedTarget - upperMotorSpeedCurrent);
```

Comment:
These calls to `putNumber` should be moved to the subsystem since we want basic information on the PID loop to always be available no matter what command is running.  For example if we have an auto command that did shooting we would want these numbers graphed without having to duplicate the code from this `TeleopShooterCommand`